kola/tests/misc/grub.go

```diff
@@ -140,8 +140,22 @@ var (
 
 coreos:
   units:
-    - name: update-engine.service
-      mask: true
+    - name: stop-and-mask-update-engine.service
+      enabled: true
+      contents: |
+        [Unit]
+        Description=Stop Update Engine
+        Conflicts=update-engine.service
+        Before=update-engine.service
+
+        [Service]
+        # This is probably redundant, since Conflict clause above
+        # should stop update engine, but eh, better safe than sorry.
+        ExecStartPre=-/usr/bin/systemctl stop update-engine.service
+        ExecStart=/usr/bin/systemctl mask update-engine.service
```

Comment:
```suggestion
        ExecStart=/usr/bin/systemctl mask --now update-engine.service
```
This also stops the service if running